README.md

```diff
@@ -9,16 +9,39 @@ Product image and description. Maybe add aside with customer endorsement.
 ## Footer Section
 Fake store address, social media links, etc.
 
-
```

Comment:
I've tried to delete this so many times and it doesn't show up in my VS but its here so if someone can please tell me how to fix it I would appreciate it! Thanks in advance!